desktop/libs/azure/src/azure/abfs/__init__.py

```diff
@@ -76,7 +75,8 @@ def strip_scheme(path):
     filesystem, file_path = parse_uri(path)[:2]
   except:
     return path
-  assert_not_equal(filesystem, '', 'File System must be Specified')
+  #@Todo@ to check the use of assert here
+  assert filesystem != '', 'File System must be Specified'
```

Comment:
This doesn't make any sense to me, it should probably be a raised exception (depending on how assert_not_equal) worked in the past.